test/test_tdvp.jl

```diff
@@ -393,11 +399,13 @@ end
 
   Sz2_step = results(step_obs)["Sz"]
   En2_step = results(step_obs)["En"]
+  infos_step = results(step_obs)["info"]
 
   @test Sz1 ≈ Sz2
   @test En1 ≈ En2
   @test Sz1 ≈ Sz2_step
   @test En1 ≈ En2_step
+  @test isnothing(findfirst(x -> x.converged != 1, infos_step)) #info.converged should always be 1
```

Comment:
You can use `all(isone, infos_step)`.

Reply:
done

Reply:
I don't see this line changed, did you push this change?

Reply:
Oops never mind, the Github interface was confusing.